<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PenYinParamWnd</class>
 <widget class="QMainWindow" name="PenYinParamWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PenYinParam</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>501</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>喷印方向</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>喷印算法</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>线缆字体</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>反视喷印</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>反向喷印</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>正向</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>反向</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>颠倒正向</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>颠倒反向</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>自动</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>打开</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>关闭</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox_4">
       <item>
        <property name="text">
         <string>横向</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>纵向</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_5">
       <item>
        <property name="text">
         <string>打开</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>关闭</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>正向延时</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>反向延时</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>喷印间隔</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>物件检测滤波</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>喷印触发</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QComboBox" name="comboBox_10">
       <item>
        <property name="text">
         <string>手动单次</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>手动连续</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>触发单次</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>触发连续</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>501</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>喷印速度测定        </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>喷印速度</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>同步信号分频</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="comboBox_11">
       <item>
        <property name="text">
         <string>定速</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>同步器</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>传感器测速</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="comboBox_6">
       <item>
        <property name="text">
         <string>------------</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>重复喷印            </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QDialogButtonBox" name="penyinbutton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>450</y>
      <width>156</width>
      <height>23</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
